nir/ir.py

```diff
@@ -100,7 +109,7 @@ class CubaLIF(NIRNode):
     r: np.ndarray  # Resistance
     v_leak: np.ndarray  # Leak voltage
     v_threshold: np.ndarray  # Firing threshold
-    w_in: np.ndarray = 1.  # Input current weight
+    w_in: np.ndarray = 1.0  # Input current weight
```

Comment:
This default value should be an np array.

Reply:
Addressed in #21